integration_test/third_party_apps_data/applications/oracledb/enable

```diff
@@ -0,0 +1,40 @@
+# Configures Ops Agent to collect telemetry from the app and restart Ops Agent.
+
+set -e
+
+# Create a back up of the existing file so existing configurations are not lost.
+sudo cp /etc/google-cloud-ops-agent/config.yaml /etc/google-cloud-ops-agent/config.yaml.bak
+
+# Configure the Ops Agent.
+sudo tee /etc/google-cloud-ops-agent/config.yaml > /dev/null << EOF
+logging:
+  receivers:
+    oracledb_audit:
+      type: oracledb_audit
+      include_paths: [/opt/oracle/admin/*/adump/*.aud]
```

Comment:
Can we remove `include_paths` here (and on line 17) to let the defaults take over so that we can test a default installation?

Reply:
I'm not 100% sure that the image we've got set up has ORACLE_HOME set appropriately, but will try it in kokoro and see what I can do.

Reply:
ORACLE_HOME's not available in our integration test set up, starting a conversation about which direction to go external to github.